packages/jobs/lib/activities.ts

```diff
@@ -505,41 +508,17 @@ export async function reportFailure(
 
 export async function cancelActivity(workflowArguments: InitialSyncArgs | ContinuousSyncArgs): Promise<void> {
     try {
-        const { syncId, syncName, nangoConnection, debug } = workflowArguments;
-
-        const context: Context = Context.current();
+        const { syncId, nangoConnection } = workflowArguments;
 
         const environmentId = nangoConnection?.environment_id;
 
-        const providerConfig: ProviderConfig = (await configService.getProviderConfig(nangoConnection?.provider_config_key, environmentId)) as ProviderConfig;
-
-        const lastSyncDate = await getLastSyncDate(syncId);
-        const syncType = lastSyncDate ? SyncType.INCREMENTAL : SyncType.INITIAL;
-
-        const syncRun = new syncRunService({
-            bigQueryClient,
-            integrationService,
-            recordsService,
-            orchestratorClient,
-            logContextGetter,
-            writeToDb: true,
-            syncId,
-            nangoConnection,
-            syncType,
-            syncName,
-            activityLogId: undefined,
-            provider: providerConfig.provider,
-            temporalContext: context,
-            debug: Boolean(debug)
-        });
-
         if ('syncJobId' in workflowArguments) {
             await updateSyncJobStatus(workflowArguments.syncJobId, SyncStatus.STOPPED);
         } else {
             await updateLatestJobSyncStatus(workflowArguments.syncId, SyncStatus.STOPPED);
         }
 
-        await syncRun.cancel();
```

Comment:
I removed the instantiation of syncRunService here because we were not using any of the required parameters, seems like unnecessary overhead. 
Let met know if that's alright